kolibri/core/auth/test/test_models.py

```diff
@@ -662,6 +663,28 @@ def test_default_facility_returns_none_when_no_settings(self):
 
 
 class FacilityUserTestCase(TestCase):
+    def setUp(self):
+        self.facility = Facility.objects.create(name="My Facility")
```

Comment:
I think it makes sense to create the facility in a setup function when that facility is expected to be reused, but in this case, it should be fine to follow the pattern of the other functions below and use the line `self.facility = Facility.objects.create()` directly within `test_all_objects_manager_returns_all_users()`